### DISCUSSION TITLE
Integration with Home Assistant

### DISCUSSION THREAD

**Discussion by razmiy** [obiwan007/despresso#120] [General]
Would it be possible to do that now similar like we do with advanced_rest_api?

**Comment by randomcoffeesnob** [comment]
Thanks for bringing up my plugin 😃 

There is an even better solution here. You can add it with mqtt.
In the wiki you find a first configuration option: 

- https://github.com/obiwan007/despresso/wiki/FAQ#how-do-i-get-my-de1-state-in-home-assistant
- https://github.com/obiwan007/despresso/wiki/Web-Page-and-REST-API#rest-api

I can add more documentation later today.

**Reply by razmiy** [reply]
Thanks a million. Appreciate your work 🙏
